### ISSUE TITLE
Clean up temporary files in test suite

### ISSUE THREAD

**Issue by charleskawczynski** [CliMA/ClimaLSM.jl#199]
## Describe the bug
Temporary files are created when running the test suite, and it would be nice if these files were cleaned up after running.

**Comment by kmdeck** [comment]
agreed, thanks for pointing this out! It also happens in our experiments folder. Ill assign myself (and get it to...sometime this sprint)

**Comment by juliasloan25** [comment]
@kmdeck has this been fixed? I think it has...

**Comment by kmdeck** [comment]
I think so too!
